diff --git a/node_modules/ssh2/lib/Channel.js b/node_modules/ssh2/lib/Channel.js
index 0120779..afe8bd7 100644
--- a/node_modules/ssh2/lib/Channel.js
+++ b/node_modules/ssh2/lib/Channel.js
@@ -12,7 +12,7 @@ const {
 const { bufferSlice } = require('./protocol/utils.js');

 const PACKET_SIZE = 32 * 1024;
-const MAX_WINDOW = 2 * 1024 * 1024;
+const MAX_WINDOW = 16 * 1024 * 1024;
 const WINDOW_THRESHOLD = MAX_WINDOW / 2;

 class ClientStderr extends ReadableStream {
